internal/idler/user_idler.go

```diff
@@ -112,10 +112,12 @@ func (idler *UserIdler) checkIdle() error {
 			return err
 		}
 		if enabled {
-			idler.doIdle()
+			err := idler.doIdle()
```

Comment:
lgtm but just occurred to me now that the `idler.user.IdlerStatus` can actually be set by `doUnIdle` or `Idle` - right? Setting the status shouldn't be the responsibility of the caller, should it? 

Approving the PR ...